examples/skrest_js_example.ml

```diff
@@ -0,0 +1,16 @@
+open Skrest_js
+
+let ( let* ) = Lwt.bind
+
+let () =
+  (let uri = Uri.of_string "https://httpbin.org/anything" in
+   let* resp = get ~follow:0 uri in
+   match resp with
+   | Ok str ->
+     print_endline str;
+     Lwt.return_unit
+   | Error err ->
+     Format.printf "Error: %a" pp_error err;
+     Lwt.return_unit
```

Comment:
Can you confirm that this works? I don't know if lwt is very happy in the browser

to test overseer, just run `dune build`, `cd _build/default/sites/overseer/src/`, `python -m SimpleHTTPServer 9000`, and navigate to `localhost:9000`, if the console doesnt give you a "runtime not found" error, you should be good

Reply:
(Actually, now that i see the jsoo lwt calls, I'm assuming that they actually DO work, and this looks great)